### PR TITLE
fixes #614 

### LINKED ISSUE #614: Use const declarations instead of var wherever constants are declared
We have established with #613 that `const` works with all the browsers we support, and it improves readability as well as (presumably) improving browser/app memory management (likely a very tiny effect).

### PR DESCRIPTION
**Pull request by TaniaMalhotra**
Changed var declarations for constants to const
Improves readability as well as browser/app memory management.

### CODE REVIEW

**Comment by Jaifroid** [comment]
Thanks for this PR @TaniaMalhotra . I'll test shortly.

**Comment by Jaifroid** [comment]
PS Tests are failing because you're an external contributor. I'll push a minor commit the branch to check tests.

**Comment by Jaifroid** [comment]
Tests now passed. @TaniaMalhotra do you see a green "Squash and merge" button below? If so, you could do the squash and merge with that button. But it may be that you don't have the right access to do it. Let me know if not.

**Comment by kelson42** [comment]
@Jaifroid I took on my ton invite @TaniaMalhotra in the Team.

Regarding the CI problem with external contributions, I can say:
* During a few months we have achieved to avoid that problem on Kiwix Android with TravisCI (I don't know the details).
* Migrating to Github Actions (what we have done in most of our repos meanwhile) has solved that problem as well.

**Comment by Jaifroid** [comment]
@kelson42 Thank you! We have #590 to start adding some tests using GitHub actions. I agree it would be good to move to that CI. For now I have a workaround which is to submit a PR of the same git commit (once code is ready for merging), and that triggers correct testing on the external contributor's PR.

File: service-worker.js [hunk 1]
```diff
@@ -28,7 +28,7 @@
  * The value is defined in app.js and will be passed to Service Worker on initialization (to avoid duplication)
  * @type {String}
  */
-var CACHE_NAME;
+const CACHE_NAME;
```
Comment (mossroy) [inline service-worker.js:31]: This can't be a constant, as this variable is assigned a value, later in this same file

Comment (Jaifroid) [reply service-worker.js:31]: @mossroy You're right. I missed that when reviewing. Oddly it didn't throw an error when I tested SW mode. I guess browsers are not treating const strictly.

**Comment by Jaifroid** [comment]
@TaniaMalhotra You will notice that a minor (but important) change is needed as @mossroy stated. When you make this change, could you please try to do it to this PR, rather than closing this and starting a new one? It keeps comments together in case anyone wants to refer back to PR discussions. Thank you!

**Comment by TaniaMalhotra** [comment]
@Jaifroid I have made a change to this PR. I'm extremely sorry for the inconvenience last time. 😥 
@mossroy @Jaifroid Thankyou for reviewing my pull request

**Comment by Jaifroid** [comment]
@TaniaMalhotra Thank you and don't be sorry - that's what review is all about!

**Comment by Jaifroid** [comment]
OK @TaniaMalhotra Tests have passed, so you can squash and merge now using the button below.

**Comment by Jaifroid** [comment]
@TaniaMalhotra Just a quick note that when you merge, we prefer that you squash all your commits into a single commit before merging. That's why I suggested you use the Squash and merge button rather than merge from your repo, because if you use the button it automatically squashes all your commits into one and asks you for a title. It keeps the master branch much neater, as we should only have one commit per PR. This is just friendly advice for future contributions!

**Comment by TaniaMalhotra** [comment]
Sure sir
Will keep in mind from the next time

**Pull request merged**